gradle.properties

```diff
@@ -25,8 +25,8 @@ mavenGroupId         = com.microsoft.graph
 mavenArtifactId      = microsoft-graph-core
 mavenMajorVersion    = 3
 mavenMinorVersion    = 0
-mavenPatchVersion    = 9
-mavenArtifactSuffix = 
+mavenPatchVersion    = 10
+mavenArtifactSuffix = RC
```

Comment:
I don't think we need a suffix here?

Reply:
Wasn't sure if this would be a RC as well since we also never announced an official release, but if this is mostly used with the SDK then RC suffix on the SDK library will likely suffice.

Reply:
will the RC only ever be on the snapshot feed? if so, I don't think we need a suffix